src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.Help.cs

```diff
@@ -16,8 +17,7 @@ public IEnumerable<Action<HelpContext>> CustomHelpLayout()
         {
             yield return (context) =>
             {
-                var originalOutputColor = Console.ForegroundColor;
-                Console.WriteLine("Waiting for options and extensions...");
+                Console.WriteLine(LocalizableStrings.HelpWaitingForOptionsAndExtensions);
```

Comment:
Don't we want to also use the output service here?

Reply:
No. The output service is not ready yet, and I would have to put that message quite a bit later.

Reply:
Thanks!